src/czbenchmarks/tasks/single_cell/k562_perturbation.py

```diff
@@ -0,0 +1,288 @@
+from typing import List, Dict
+import pandas as pd
+import numpy as np
+import logging
+from ..task import Task, TaskInput, TaskOutput
+from ...tasks.types import CellRepresentation
+from ...metrics import metrics_registry
+from ...metrics.types import MetricResult, MetricType
+from ...constants import RANDOM_SEED
+
+from tqdm import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+def binarize_values(y_true, y_pred):
+    ids = np.where(~np.isnan(y_true) & ~np.isinf(y_true))[0]
+    y_true = y_true[ids]
+    y_pred = y_pred[ids]
+    pred_binary = (y_pred > 0).astype(int)
+    true_binary = (y_true > 0).astype(int)
+    return true_binary, pred_binary
+
+
+class K562PerturbationTaskInput(TaskInput):
+    """Pydantic model for PerturbationTask inputs."""
+
+    de_res_wilcoxon_df: pd.DataFrame
+    pred_df: pd.DataFrame
+
+
+class K562PerturbationOutput(TaskOutput):
+    """Output for perturbation task."""
+
+    pred_log_fc_dict: Dict[str, np.ndarray]
+    true_log_fc_dict: Dict[str, np.ndarray]
+
+
+class K562PerturbationTask(Task):
```

Comment:
This task will be used on other datasets, so the name shouldn't be specific to Replogle or K562 cells. Maybe something like "PerturbationExpressionPrediction"? Or something even better than you think of. Will need to update the pydantic classes, example script, etc.